.gitignore

```diff
@@ -29,3 +29,4 @@ tmp-*
 combine_logger.out
 docker_stderror
 .ipynb_checkpoints/
+.idea
```

Comment:
is this needed ?

Reply:
At least for my IDE as this stupid folder is always created. Is it that bad?

Reply:
i was just wondering, is fine by me :)